CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **BREAKING**: Removed `workdir`, `loaders`, `default_loader`, and `save_file_encoding` fields from `FileManager` and added `file_manager_driver`.
 - **BREADKING**: Removed `mime_type` field from `ImageArtifact`. `mime_type` is now a property constructed using the Artifact type and `format` field.
 - Moved [Griptape Docs](https://github.com/griptape-ai/griptape-docs) to this repository.
+- Bumped `pinecone` from `^2` to `^3`.
```

Comment:
could this be a breaking change if people are using the old version of pinecone? or is this fully internal to Griptape implementation?

Reply:
I guess it is _technically_ breaking if users are using the client natively. But if users are using it through Griptape there'd be no changes required. Will escalate to breaking just as a safety measure.